modules/cluster-compare-patching.adoc

```diff
@@ -0,0 +1,190 @@
+:_mod-docs-content-type: PROCEDURE
+[id="cluster-compare-patching_{context}"]
+= Patching a reference configuration
+
+In certain scenarios, you might need to patch the reference configuration to handle expected deviations in a cluster configuration. The plugin applies the patch during the comparison process, modifying the specified resource fields as defined in the patch file.
+
+For example, you might need to temporarily patch a template because a cluster uses a deprecated field that is out-of-date with the latest reference configuration. Patched files are reported in the comparison output summary.
+
+You can create a patch file in two ways:
+
+* Use the `cluster-compare` plugin to generate a patch YAML file. 
+
+* Create your own patch file.
+
```

Comment:
Does patching only work with local files? 

This produced a patch file: 
`oc cluster-compare -r metadata.yaml -o 'generate-patches' --override-reason "A valid reason for the override" --generate-override-for "optional/local-storage-operator/StorageClass.yaml" > patch`

This didn't:
`oc cluster-compare -r https://raw.githubusercontent.com/openshift-kni/cnf-features-deploy/master/ztp/kube-compare-reference/metadata.yaml -o 'generate-patches' --override-reason "A valid reason for the override" --generate-override-for "https://github.com/openshift-kni/cnf-features-deploy/master/ztp/kube-compare-reference/optional/local-storage-operator/StorageClass.yaml"`

Reply:
That's a great question!  I think when referring to the CR via `--generate-override-for` it expects the relative path, not the absolute path... So in the 2nd example if you set `optional/local-storage-operator/StorageClass.yaml` I think it would work?

Reply:
Yeap a relative path worked, thanks! I'll update for next draft.

Reply:
I added a note about using a relative path.